### PR TITLE
[GCI 2019] [Task] Add Screenshots

### PR DESCRIPTION
**Pull request by trungnt2910**
I have added a screenshot, as well as updating Readme.txt to README.md to include the new Screenshot for ExpressionEvaluator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,40 @@
+# Expression Evaluator 
+
+© 2000 Chris Dragan
 
-Expression Evaluator © 2000 Chris Dragan
 http://chrisdragan.org
```
Comment (humdingerb) [inline README.md]: Remove, the URL seems to be dead.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
 
+![Screenshot](Screenshot.png)
+
 Usage:
```
Comment (humdingerb) [inline README.md]: Use ## (or ###, we don't want it to be too big) to format as a sub-heading.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,40 @@
...
-	- binary,
-	- octal.
+- integer decimal,
+- floating-point decimal,
+- floating-point exponential,
+- hexadecimal,
+- binary,
+- octal.
 
 Supported operators:
```
Comment (humdingerb) [inline README.md]: `##` or `###` again for a sub-heading.

File: README.md [hunk 4]
```diff
@@ -1,30 +1,40 @@
...
-+ - * / % ^ << >> & | ~ && || ^ ~ == = != <> > >= < <=
+
+	\+ \- \* / % ^ << >> & \| ~ && \|\| ^ ~ == = \!= <> > >= < <=
+
 (note: ^ is power and not xor).
 
 Supported functions:
```
Comment (humdingerb) [inline README.md]: `##` or `###` again for a sub-heading.

File: README.md [hunk 5]
```diff
@@ -1,30 +1,40 @@
...
+
 abs, neg, sin, cos, tan (tg), ctan (ctg), asin (arcsin), acos (arccos), atan (arctan, atg, arctg), actan (arcctan, actg, arcctg), sqr, sqrt, exp, log (base 2), lg (base 10), ln (base e)
 
 Supported constants:
```
Comment (humdingerb) [inline README.md]: `##` or `###` again for a sub-heading.

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (trungnt2910) [reply README.md]: Sorry, I wasn't familiar with Markdown at that time, so I didn't know that.

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**